README.md

```diff
@@ -17,31 +17,31 @@ Please see `docs` directory for additional endpoint examples
 package main
 
 import (
-	"context"
-	"fmt"
-	"github.com/mxenabled/atrium-go"
-	"github.com/antihax/optional"
+  "context"
```

Comment:
Looks like you're undoing @minond 's change here to using tabs instead of spaces.